xgboost_synthetic/build-train-deploy.ipynb

```diff
@@ -27,22 +27,7 @@
    "cell_type": "markdown",
```

Comment:
<p>For current master, you don't need to run this <code>notebook_setup()</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/examples/pull/662/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Shall we keep it for `requirements.txt` install?